encoder-brave/src/main/java/zipkin2/reporter/otel/brave/SpanTranslator.java

```diff
@@ -154,11 +157,19 @@ private Span.Builder builderForSingleSpan(MutableSpan span, ResourceSpans.Builde
     resourceBuilder.addAttributes(stringAttribute(SemanticConventionsAttributes.TELEMETRY_SDK_LANGUAGE, TELEMETRY_SDK_LANGUAGE))
         .addAttributes(stringAttribute(SemanticConventionsAttributes.TELEMETRY_SDK_NAME, TELEMETRY_SDK_NAME))
         .addAttributes(stringAttribute(SemanticConventionsAttributes.TELEMETRY_SDK_VERSION, TELEMETRY_SDK_VERSION));
-
-    maybeAddStringAttribute(spanBuilder, SemanticConventionsAttributes.NETWORK_LOCAL_ADDRESS, span.localIp());
-    maybeAddIntAttribute(spanBuilder, SemanticConventionsAttributes.NETWORK_LOCAL_PORT, span.localPort());
-    maybeAddStringAttribute(spanBuilder, SemanticConventionsAttributes.NETWORK_PEER_ADDRESS, span.remoteIp());
-    maybeAddIntAttribute(spanBuilder, SemanticConventionsAttributes.NETWORK_PEER_PORT, span.remotePort());
+    if (span.kind() == Kind.SERVER || span.kind() == Kind.CONSUMER) {
+      resolveAddressAndPort(span, span.localIp(), span.localPort(), (address, port) -> {
```

Comment:
last style suggestion. If you notice now, the contents of the lambda passed are the same in both cases. So, maybe you can change the function to `setServerAddressAndPort` and don't use a lambda as the last arg. wdyt?

Reply:
Mostly addressed in https://github.com/openzipkin-contrib/zipkin-otel/commit/70e673fe952ae00178b8234c1a96216b844cbd43
Renamed the method in https://github.com/openzipkin-contrib/zipkin-otel/pull/28/commits/0feaba63a4d01807e1922a7eab8da363d5477d77